test/time_evolvers/test_pvqd.py

```diff
@@ -48,7 +48,7 @@ class WhatAmI(Gate):
     def __init__(self, angle):
         super().__init__(name="whatami", num_qubits=2, params=[angle])
 
-    def inverse(self):
+    def inverse(self, annotated: bool = False):
```

Comment:
This probably deserves a comment that can be removed once Qiskit 0.46 reaches eol.

Reply:
After a quick look at https://github.com/Qiskit/qiskit/pull/11593 it looks like that argument is intended to live in 1.0+, so the comment wouldn't be necessary, right?

Reply:
That argument is new in 1.0 - and while adding it now is compatible with 0.46 as it was without causes lint to fail as you can see in the CI jobs which have recently been merged where the monitor jobs failed (they are not required to pass to merge)
```
************* Module test.time_evolvers.test_pvqd
test/time_evolvers/test_pvqd.py:51:4: W0221: Number of parameters was 2 in 'Instruction.inverse' and is now 1 in overriding 'WhatAmI.inverse' method (arguments-differ)
```
And this is just in a test case to not main code

Reply:
Ok, then I think that the comment is not necessary here.